mantidproject/mantid
# Add Step Scan Interface to Workbench

**Description of work.**

This adds the [Step Scan Interface](https://www.mantidproject.org/Step_Scan_Interface) to the mantid workbench. We modified the existing code to build with Qt5 and made the necessary changes to connect the instrument view and generate figures.

**Report to:** @AndreiSavici 

**To test:**



See acceptance criteria in issue blow.

Fixes https://code.ornl.gov/sns-hfir-scse/spectroscopy/spectroscopy/-/issues/21

Add 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
